### ISSUE TITLE
multiple versions of prosemirror-model after updating to 1.17.0

### ISSUE THREAD

**Issue by thegnuu** [ProseMirror/prosemirror#1282]
I updated `prosemirror-model` and all other `prosemirror-*` deps in my repository to the newest version (see below) and since the update, I receive the following error when trying to type or add content programmatically:

`Can not convert <" "> to a Fragment (looks like multiple versions of prosemirror-model were loaded)`

Current prosemirror modules:

```
"prosemirror-commands": "^1.3.0",
"prosemirror-history": "^1.3.0",
"prosemirror-keymap": "^1.2.0",
"prosemirror-model": "^1.17.0",
"prosemirror-schema-list": "^1.2.0",
"prosemirror-state": "^1.4.0",
"prosemirror-view": "^1.25.0",
```

I already deleted my `yarn.lock` and `node_modules` which helped if I got this error on previous versions.

Has anyone an idea how to fix this issue, if reinstalling the modules won't help?
I would like to stay on `1.17.0` if possible since this release finally includes TypeScript types :)

Thanks for this awesome toolset and your help!

**Comment by marijnh** [comment]
Use `npm ls -a` to figure out which versions are installed and why. Maybe something, somewhere, is specifying too narrow a version range.

**Comment by marijnh** [comment]
(Also consider dropping yarn, npm is as good now and more well-maintained.)

**Comment by thegnuu** [comment]
@marijnh Thank you for your response! 

I already checked the installed versions and everything looked fine to me... That's why I opened the issue, in the past I was easily able to resolve this issue myself ;)

But after hours of debugging I finally found the issue.

In my setup I have an `sdk` lib which is imported into our Angular applications. It looks like one of my exports was breaking the `instanceof` check in prosemirror. I was exporting a base schema (`export const baseSchema = new Schema({...})`) which was used in the `ui` lib, if I create this base schema directly in the `ui` lib and not export it from our `sdk` everything works again....

I honestly don't know why removing the `@types/prosemirror-*` and switching to the newest `prosemirror-*` versions broke it since there were no other changes. Looking at it it should have not worked in the first place prior to this though, but anyways.

Regarding switching to npm, you are right. But I am currently to lazy and busy to update all of our 90 repos, tools and automatisations, and I don't like the idea of switching tools based on the workspace ;)

Thanks!
